web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -17,92 +17,42 @@
  */
 
 import React, { lazy, Suspense, useState } from 'react';
-import styled from 'styled-components';
-import { Flex, Text, TopNav } from 'design';
+import styled, { useTheme } from 'styled-components';
+import { Link } from 'react-router-dom';
+import { Flex, Image, Text, TopNav } from 'design';
 
 import { matchPath, useHistory } from 'react-router';
 
 import { BrainIcon } from 'design/SVGIcon';
 
-import { ArrowLeft } from 'design/Icon';
+import { ArrowLeft, ChevronRight, SlidersVertical } from 'design/Icon';
+import { HoverTooltip } from 'shared/components/ToolTip';
 
 import useTeleport from 'teleport/useTeleport';
-import useStickyClusterId from 'teleport/useStickyClusterId';
 import { UserMenuNav } from 'teleport/components/UserMenuNav';
 import { useFeatures } from 'teleport/FeaturesContext';
-
-import cfg from 'teleport/config';
+import { NavigationCategory } from 'teleport/Navigation/categories';
 
 import { useLayout } from 'teleport/Main/LayoutContext';
-import { KeysEnum } from 'teleport/services/storageService';
 import { getFirstRouteForCategory } from 'teleport/Navigation/Navigation';
 
-import ClusterSelector from './ClusterSelector';
 import { Notifications } from './Notifications';
 import { ButtonIconContainer } from './Shared';
+import logoLight from './logoLight.svg';
+import logoDark from './logoDark.svg';
 
 const Assist = lazy(() => import('teleport/Assist'));
 
-export function TopBar() {
+export function TopBar({ navigationItems }: NavigationProps) {
```

Comment:
Can `navigationItems` come from `useFeatures()` instead of being passed through to here?

Reply:
I sorta hijacked `topmenuitem` since it fit and my filter for topmenu items is if it's defined and a `Resource` category. worked well, and also was able to use the already filtered items based on access